frontend/packages/ux-editor/src/components/Properties/Properties.tsx

```diff
@@ -24,6 +24,9 @@ export const Properties = () => {
     }
   };
 
+  const isSubForm = formItem && formItem.type === 'SubForm';
+  const hasSubformAndLayoutSet = isSubForm && !!formItem.layoutSet;
+
   return (
     <div className={classes.root} key={formItemId}>
       {!formItem ? (
```

Comment:
I liked it 😊, 
but i have 2 things 🙂here : 
- regarding the name `renderAccordions  `I think it does not reflect the content of the logic here:
 `formItem.type !== "Subform" || !!formItem.layoutSet;`   and here: 
```
  {renderAccordions && (
             <Accordion color='subtle'>  
```
It will be interpreted like if `renderAccordion `then render Accordion , what about this name `isNotSubFormOrHasLayoutSet`  ? 🙂
- regadign the name `SubForm` we cannt change it to `Subform `because it is actually coming from the `enum `in `componentType.ts ` , If so, then we need to change it in this component also and then we should also change it in the other components that use it. what do you think? 🙂

Reply:
I changed it to `isNotSubformOrHasLayoutSet` 🙂